test/test_prototype_models.py

```diff
@@ -117,13 +119,22 @@ def test_video_model(model_fn, dev):
     TM.test_video_model(model_fn, dev)
 
 
+@needs_cuda
+@pytest.mark.parametrize("model_builder", (models.optical_flow.raft_large, models.optical_flow.raft_small))
```

Comment:
Why not the following?
```suggestion
@pytest.mark.parametrize("model_builder", TM.get_models_from_module(models.optical_flow))
```